src/utils/deepClone.ts

```diff
@@ -0,0 +1,25 @@
+// ==LICENSE-BEGIN==
+// Copyright 2017 European Digital Reading Lab. All rights reserved.
+// Licensed to the Readium Foundation under one or more contributor license agreements.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file exposed on Github (readium) in the project repository.
+// ==LICENSE-END==
+
+const fromEntries = (it: Array<[string, any]>): any =>
+    it.reduce<any>(
+        (pv, [_key, _value]) =>
+            pv[_key] = typeof _value === "object" ? fromEntries(_value) : _value,
+        {});
+
+const ObjFromEntries = Object.fromEntries || fromEntries;
+
+export const deepClone = (o: unknown): any => {
+
+    const _clone = (_o: unknown): Array<[string, unknown]> =>
+        Object.entries(_o).map<[string, unknown]>(
```

Comment:
If `_value` is `null`, the program crashes.

`node -e "console.log(Object.entries(null))"`

(also tested with actual `deepClone()` call)

Reply:
thank you